components/admin_console/admin_definition.jsx

```diff
@@ -250,6 +252,20 @@ const usesLegacyOauth = (config, state, license, enterpriseReady, consoleAccess,
     )(config, state, license, enterpriseReady, consoleAccess, cloud);
 };
 
+const getRestrictedIndicator = (displayBlocked = false) => ({
+    value: (cloud) => (
+        <RestrictedIndicator
+            useModal={false}
+            blocked={displayBlocked || cloud?.subscription?.is_free_trial !== 'true'}
```

Comment:
What does `blocked` do? It seems we are setting this to any cloud subscription regardless of license that is not in a free trial?

Reply:
`blocked` controls the indicator type to show: lock key or high voltage.
Updated.